saleor/graphql/order/types.py

```diff
@@ -1031,6 +1046,17 @@ def wrap_shipping_method_with_channel_context(data):
 
     @staticmethod
     @traced_resolver
+    def resolve_delivery_method(root: models.Order, info):
+        if root.shipping_method_id:
+            return Order.resolve_shipping_method(root, info)
+        if root.collection_point_id:
+            collection_point = WarehouseByIdLoader(info.context).load(
+                root.collection_point_id
+            )
+            return collection_point
+        return None
+
+    @staticmethod
     # TODO: We should optimize it in/after PR#5819
     def resolve_available_shipping_methods(root: models.Order, info):
```

Comment:
This method had wrapper `traced_resolver` but it is missing now. Is it expected?

Reply:
Good point. Done.